inst/extdata/temp-data-inject/L201.nonghg_steepness.csv

```diff
@@ -1,7 +1,7 @@
-INPUT_TABLE
-Variable ID
-GDPCtrlSteep
-
+# INPUT_TABLE
+# Variable ID
+# GDPCtrlSteep
+#
```

Comment:
I assume that this file will be produced by another persons chunk.  It is an input file for me, and has been a pain to work with.  I intensely dislike the L201* file.  All that being said, should I make edits to this input file if it is going to be rewritten by someone else?